driver-core/src/main/com/mongodb/connection/netty/NettyStreamFactory.java

```diff
@@ -47,7 +47,7 @@
      * @param eventLoopGroup the event loop group that all channels created by this factory will be a part of
      * @param allocator the allocator to use for ByteBuf instances
      */
-    public NettyStreamFactory(final SocketSettings settings, final SslSettings sslSettings, final NioEventLoopGroup eventLoopGroup,
+    public NettyStreamFactory(final SocketSettings settings, final SslSettings sslSettings, final EventLoopGroup eventLoopGroup,
```

Comment:
This is backwards breaking but better to do it now when no one is likely using it yet.